dali/base/dadfs.cpp

```diff
@@ -3295,6 +3295,22 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
     virtual void setAccessedTime(const CDateTime &dt) = 0;                      // set date and time last accessed
     virtual bool isExternal() const { return external; }
 
+    virtual void getExpirationInfo(int &expireDays, StringBuffer &expirationDate)
```

Comment:
It would be better to have a single expiry related method (this new one duplicates the existing functionality of getExpiry()).
I suggest adding StringBuffer *expirationData as an optional parameter to the existing getExpiry()